### DISCUSSION TITLE
Pump recommendation

### DISCUSSION THREAD

**Discussion by YBonline** [aqualinkd/AqualinkD#357] [Q&A]
So I bought a new home earlier this year, and it has an Aqualink RS control panel with a Jandy VSPlusHP pump (model VSPHP270AUT). I wasn't thrilled with the Aqualink app and wanted better HA integration, and also was not thrillled with the pump (my previous home had a Pentair VSF pump that I liked much better, one of the main issues being the pool guy can't control the speed of the Jandy.

Anyways, right when I was about to setup this project to try to solve some of my issues, I got hit by a hurricane and my pump was flooded. The RS control panel still seems to work, the pump does not seem to work, and I'm not sure on the heater (it doesn't blow the breaker) or salt cell yet, so now I'm trying to decide what to do.

Should I stick with a Jandy VS pump, or will I have more options such as the variable speed flow adjustment and better controls on a Pentair pump, and any recommendation on the model? I'd also like to add pH/ORP sensors while I'm having to repipe some stuff anyways... Any advise for the ideal HA pool automation setup before I go start spending money on fixing this mess is much appreciated

**Comment by sfeakes** [comment]
The version # of the AqualinkRS panel is the most import thing to know before anyone can advise you.
There is a wide range of things you can / can’t do / control depending on the version.

in general the Pentair VS/F pumps provide a lot more information than Jandy VSP and all can be fully controlled from a Jandy RS panel (depending on version).  VS vs VF vs VSF will really depend on how you want to control it, Ie using a RPM or GPM setting. 

PH/ORP sensors integrated to the RS panel directly will need the latest RS panel, going back a few versions you can get those readings using Jandy chemical feeder, but they are stupid $$$$$.  AqualinkD allows you to integrate any PH/OPH sensor readings using 3rd party sw.

AqualinkD is fully integrated into homeassistant so no need to worry their.

**Reply by YBonline** [reply]
How do you know the version of the RS panel? I don't see it written anywhere on it... The serial number starts with H18RSLML #### and below that it says H0263500C? Does that tell anything?

**Comment by sfeakes** [comment]
As a last resort you can get the version from the circuit board.  But you can also get it from whatever Jandy keypad you have. (All button, onetouch, iaqualinktouch etc). Depends on which device you have as to how to read the version.  Usually under some info screen
